lib/fluent/plugin/out_newrelic.rb

```diff
@@ -147,7 +147,7 @@ def get_compressed_payloads(logs)
           second_half = get_compressed_payloads(logs.slice(midpoint, logs.length))
           return first_half + second_half
         else
-          log.error("Can't compress record below required maximum packet size and it will be discarded. Record: #{logs[0]}")
+          log.error("Can't compress record below required maximum packet size and it will be discarded.")
```

Comment:
Hey @jcsobrino , can we probably print the first 100 characters of the log, and also display what is the log size in Kbs once compressed and the character count?

Reply:
Instead of print part of the offending log, we have included the log timestamp and its uncompressed/compressed size in bytes for error tracing purposes.